task/buildah-remote-oci-ta/0.1/buildah-remote-oci-ta.yaml

```diff
@@ -413,13 +427,15 @@ spec:
       chmod +x scripts/script-build.sh
       rsync -ra scripts "$SSH_HOST:$BUILD_DIR"
       ssh $SSH_ARGS "$SSH_HOST" $PORT_FORWARD podman  run $PODMAN_PORT_FORWARD \
+       --tmpfs /run/secrets \
```

Comment:
Same as https://github.com/konflux-ci/build-definitions/pull/1172#discussion_r1682659433, this breaks the ADDITIONAL_SECRET feature

Reply:
Ah, nevermind - this is a mount for the podman container that runs the build, not for the build itself